diff --git a/dist/vendor/schema-org-v3/shared/schema-org.DYFTMLZ0.mjs b/dist/vendor/schema-org-v3/shared/schema-org.DYFTMLZ0.mjs
index bc7b1e191db9bb186cef39f0d7fcbdcbe25bbcb3..6ecd92f215aea3b29b7c4066d7178cefd3891c68 100644
--- a/dist/vendor/schema-org-v3/shared/schema-org.DYFTMLZ0.mjs
+++ b/dist/vendor/schema-org-v3/shared/schema-org.DYFTMLZ0.mjs
@@ -1776,6 +1776,7 @@ function UnheadSchemaOrg(config = {}, meta = () => ({}), options) {
           }
         },
         "entries:normalize": ({ tags }) => {
+          graph = graph || createSchemaOrgGraph();
           for (const tag of tags) {
             if (tag.tag === "script" && tag.props.type === "application/ld+json" && tag.props.nodes) {
               const nodes = tag.props.nodes;
